package/kedro_viz/server.py

```diff
@@ -213,31 +244,66 @@ def commands():
     type=click.Path(dir_okay=False, writable=True),
     help="Path to save the pipeline JSON file",
 )
-def viz(host, port, browser, load_file, save_file):
+@click.option(
+    "--pipeline",
+    type=str,
+    default=None,
+    help="Name of the modular pipeline to run."
```

Comment:
```suggestion
    help="Name of the modular pipeline to run. "
```

Reply:
Do we want this for all click options?

Reply:
No, help string here is split into 2 lines, the first one lacks the space, otherwise both of them will be concatenated without a space in between the sentences.